### ISSUE TITLE
Port bindings lost on restart via list

### ISSUE THREAD

**Issue by Slamper** [crosbymichael/dockerui#78]
The port bindings of my containers are lost when I start them by selecting the checkboxes on the list and then clicking start

I'm using docker:
Client version: 1.4.1
Client API version: 1.16
Go version (client): go1.3.3
Git commit (client): 5bc2ff8
OS/Arch (client): linux/amd64
Server version: 1.4.1
Server API version: 1.16
Go version (server): go1.3.3
Git commit (server): 5bc2ff8

and the latest version of dockerUI

The HostConfig Payload is not sent when the container is started this way

Code snippet from batch()
```javascript
angular.forEach(items, function(c) {
            if (c.Checked) {
                counter = counter + 1;
                action({id: c.Id /* >>HostConfig missing here ( maybe only if action == start )<<*/ },
```

That means container info has to be fetched before the container can be started

**Comment by kevana** [comment]
Yep, we'll need to add a special case for start and restart to pass the HostConfig in. If I remember right it may get a little messy because getting a list of containers doesn't return all of their settings.

**Comment by kevana** [comment]
This should be fixed, you can test it out on the master branch.

**Comment by Slamper** [comment]
Fixed by #79
